build/x86_64_stretch/Dockerfile

```diff
@@ -26,5 +26,6 @@ RUN apt-get -y update \
         make \
         pkg-config \
         python-dev \
```

Comment:
Do you still need the above python-dev

Reply:
Yes, Python 2 libraries on Debian stretch are in the `python-dev` package.